File: src/entity/jx3Server.ts
export class Jx3Server {
	public id: number | null = null;
	public zone: string = '';
	public server: string = '';
	public status: string = '';

	constructor(zone = '', server = '', status = '') {
		this.zone = zone;
		this.server = server;
		this.status = status;
	}
}

File: src-tauri/src/entity/jx3_server.rs
use serde::{Deserialize, Serialize};

#[derive(Debug, Clone, Serialize, Deserialize)]
pub(crate) struct Jx3Server {
    pub(crate) id: Option<i32>,
    pub(crate) zone: String,
    pub(crate) server: String,
    pub(crate) status: String,
    pub(crate) last_update_time: Option<String>,
}
